train.py

```diff
@@ -44,13 +45,13 @@ def _str_to_bool(s):
 
     parser = argparse.ArgumentParser(description='WaveNet example network')
     parser.add_argument('--batch_size', type=int, default=BATCH_SIZE,
-                        help='How many wav files to process at once.')
+                        help='How many wav files to process at once. Default: 1')
     parser.add_argument('--data_dir', type=str, default=DATA_DIRECTORY,
                         help='The directory containing the VCTK corpus.')
     parser.add_argument('--store_metadata', type=bool, default=False,
                         help='Whether to store advanced debugging information '
                         '(execution time, memory consumption) for use with '
-                        'TensorBoard.')
+                        'TensorBoard. Default: False')
```

Comment:
Let's also use a variable with default parameter value here (and other similar places).